### PR TITLE
set drf dominant resource name

### PR DESCRIPTION
**Pull request by hzxuzhonghu**

### CODE REVIEW

File: pkg/scheduler/plugins/drf/drf.go [hunk 1]
```diff
@@ -155,23 +158,25 @@ func (drf *drfPlugin) OnSessionOpen(ssn *framework.Session) {
 }
 
 func (drf *drfPlugin) updateShare(attr *drfAttr) {
-	attr.share = drf.calculateShare(attr.allocated, drf.totalResource)
+	attr.dominantResource, attr.share = drf.calculateShare(attr.allocated, drf.totalResource)
```
Comment (hex108) [inline pkg/scheduler/plugins/drf/drf.go]: Where will `dominantResource` be used?

Comment (hzxuzhonghu) [reply pkg/scheduler/plugins/drf/drf.go]: should be logged

**Pull request merged**